crates/http-api-bindings/src/embedding/mod.rs

```diff
@@ -21,6 +21,15 @@ pub async fn create(config: &HttpModelConfig) -> Arc<dyn Embedding> {
                 .as_deref()
                 .expect("api_endpoint is required"),
             config.api_key.clone(),
+            false,
+        ),
+        "llama.cpp/legacy_embedding" => LlamaCppEngine::create(
```

Comment:
```suggestion
        "llama.cpp/before_b3xxxx_embedding" => LlamaCppEngine::create(
```

explicit naming of version for easier configuration.